examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,30 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for a binary, and shows
```

Comment:
s/a binary/gRPC client and server applications/ maybe?

Reply:
Yeah sure. I'll leave for a binary because the reason I had this language is the other OTel example is per client/server (Dial Option and Server Option) and this is you set it globally through CSM once per binary.